src/components/slide/slide.js

```diff
@@ -293,10 +303,17 @@ export default function Slide({
             <AnimatedDiv
               ref={setStepContainer}
               onClick={handleClick}
-              tabIndex={
-                Object.entries(frameOverrideStyle).length > 0 ? 0 : undefined
-              }
-              style={{ ...springFrameStyle, ...frameOverrideStyle }}
+              tabIndex={inOverviewMode && isActive ? 0 : undefined}
+              style={{
+                ...springFrameStyle,
+                ...frameOverrideStyle,
+                ...(inOverviewMode &&
+                  hover && {
+                    outline: '2px solid white'
```

Comment:
Added outline style on hover to create the following effect when user is selecting a slide in OVERVIEW_MODE:

![spectacle-hover-styling-overview-mode](https://user-images.githubusercontent.com/16032631/101220522-ab9b0b00-363a-11eb-81e1-3e4ae53fa1a6.gif)